docs/content/docs/ci/CIGradlePlugin.md

```diff
@@ -38,57 +38,143 @@ builds {
 - fullCheck - максимально полные проверки, можем жертвовать скоростью
 - release - набор действий необходимых для предоставления всех релизных артефактов
 
-### Steps
+## Steps
 
-Внутри сценариев описываем декларативно необходимые шаги:
+Step is a declaration to run some logic. It works inside a scenario:
```

Comment:
It's not clear that unitTests, lint, uiTests are steps

Reply:
I've commented in the example. 
I think it's enough here cause we have a detailed explanation below.